flowauth/frontend/src/UserAdminDetails.jsx

```diff
@@ -48,14 +43,16 @@ class UserAdminDetails extends React.Component {
   async componentDidMount() {
     const json = getUser(this.props.item_id);
     try {
+      console.log("Incoming JSON:", await json)
       this.setState(await json);
       this.setState({ edit_mode: true });
       if ((await json)["has_two_factor"]) {
         this.setState({ two_factor_can_be_disabled: true });
       }
+      //this.updateRoles(await json["roles"])
     } catch (err) {
       if (err.code !== 404) {
-        this.setState({ hasError: true, error: err });
+        this.setState({ hasError: true, errors: err });
```

Comment:
If you expect more than one here should it not be an array and you add to it?
If it's just one why rename it?

Reply:
I think this is actually correcting it to match the state as declared at init: https://github.com/Flowminder/FlowKit/pull/5163/files#diff-91d75c71d0bfc65946adf1de0285ade03492ef5f84f3546a32b13761b329d56aL46

But, I think the correction should be the other way around - i.e. rename errors error